yarn.lock

```diff
@@ -3614,6 +3623,92 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@fluentui/keyboard-keys@npm:^9.0.3":
+  version: 9.0.3
+  resolution: "@fluentui/keyboard-keys@npm:9.0.3"
+  dependencies:
+    "@swc/helpers": ^0.4.14
+  checksum: ab723fb9580d71f83861189cc3447d8f1500b2d76f7f71c915ab8f23591bd47d9b8626ae98dcb22f1f8e815f6c350a93553158fa465e2d40a5febd5b722a88ea
+  languageName: node
+  linkType: hard
+
+"@fluentui/priority-overflow@npm:^9.0.3":
```

Comment:
It's worth to check the bundle size impact of these transient dependencies

Reply:
This is necessary only due to how our storybook setup wraps the contents by default, which is to center everything. I don't think we need to document this. This is basic flex layout.

Reply:
Josh checked when he did the initial experiment. He said it was the size of two of our buttons. I think our code would likely be of somewhat similar size if we decided to maintain our own version.

Reply:
Have you confirmed? If so, add a line to PR description for future reference of current decision making.

Reply:
Adds 5kb minified and gzipped. https://bundlephobia.com/package/@fluentui/react-overflow@9.0.19